src/actions/MenuActions.js

```diff
@@ -3,6 +3,9 @@ import axios from 'axios';
 
 // REQUESTS
 
+let breadcrumbsCache = {};
```

Comment:
@wiadev here is my input as yet another javascript-noob :)

Loud thinking: i think the frontend side shall not cache those results at all. It shall simply call the endpoint each time user is pressing on the breadcrumb item.
Reasons:
* we are already caching it on backend side and it would be quite difficult and error prone to invalidate all caches along the road, when a sysadm is changing the menu.
* when and if we would want to optimize that part and safe some calls, we can do browser level caching (ETag, cache control etc), so basically, when the backend menu cache is invalidated we would send a different ETag.

wdyt?